File: 1) PYTHON/troco.py
precoUnitario = float(input( "Digite o preço unitário do produto: " ))
qtdProduto = float(input( "Digite a quantidade de produto comprado: " ))
dinheiro = float(input( "Quanto em dinheiro você tem, para pagar o produto: " ))
preco = precoUnitario * qtdProduto
troco = dinheiro - preco

print( f"O seu troco será de: R${ troco:.2f }" )

File: 2) JAVA/troco.java
import java.util.Scanner;

public class Main {
	public static void main( String[ ] args ) {
		Scanner scanner = new Scanner( System.in );
		float troco, preco, dinheiro;
		int quant;
		
		System.out.println( "Digite a quantidade do produto: " );
		quant = scanner.nextInt( );
		
		System.out.println( "Digite o valor do produto: " );
		preco = scanner.nextFloat( );
		
		System.out.println( "Digite o dinheiro pago: " );
		dinheiro = scanner.nextFloat( );
		
		troco = dinheiro - (quant * preco);
		
		System.out.printf( "Troco: %.2fR$", troco );
		
		scanner.close( );
	}
}